system/jlib/jlog.cpp

```diff
@@ -485,6 +485,81 @@ StringBuffer & LogMsg::toStringXML(StringBuffer & out, unsigned fields) const
     return out;
 }
 
+StringBuffer & LogMsg::toStringJSON(StringBuffer & out, unsigned fields) const
+{
+    out.ensureCapacity(LOG_MSG_FORMAT_BUFFER_LENGTH);
+    out.append("{ \"MSG\": \"").append(text.str()).append("\"");
```

Comment:
What characters need quoting/encoding?  double quote if nothing else.

Reply:
yes, the message should be encoded.